### DISCUSSION TITLE
Ableist language used in README

### DISCUSSION THREAD

**Discussion by zcolleyz** [Myndex/apca-w3#15] [General]
Just want to flag some potentially ableist language used in your README, I like the humorous parody posters you've made but the one that says "don't be a dummy" is using language that has a history of ableism. 

Calling someone 'dumb', 'dummy' as an insult is from people devaluing mute people and assuming they were less intelligent because they didn't speak.

Also you can see how we moved from 'deaf and dumb' to 'deaf-mute'. I can't claim to be an expert or speak for this community, so if there is someone then I'd be happy to hear their perspective too.

I'm aware the phrase has a double meaning because its coupled with crash test dummy but its still an insult because of the ableism.

**Comment by Myndex** [comment]
Hi @zcolleyz 
Thank you for your comment. “Dummy” refers to an inanimate object. It is not directed at any person  nor group of persons.

In the case of the poster, that was as stated in the original that we were satirizing. And it is clear that the object is a test crash dummy, with no other meaning. 

The following is my personal opinion from my point of view as a former civil rights columnist.

The meaning you are ascribing is obsolete, dating back to the 1500s, and was out of use long ago (1800s); moreover has no relevance to the subject of the poster.

If we were to attempt to “cleanse” the English language of everything that some distant etymology might be construed as “offensive”, the remaining language would be little more than a skeleton, and useless for cogent communication.

UW seems to be attempting that—I call it their NewSpeak dictionary, referring of course to Orwell’s 1984. As a former columnist on civil rights and constitutional law, I find such attempts to obfuscate the language offensive, and it is deeply concerning because that energy needs to be directed toward bad actions, not speech.

There is a bright line, and that is one of personal harassment or personal attacks, but that is more of an action, not a matter of abstract speech. It is personal harassment that should be quashed. 

The current climate of NewSpeak revisionism is notwithstanding in a healthy and open society, and per history such word-burning has unintended consequences, with no actual benefit.

Thank you for reading.

**Comment by zcolleyz** [comment]
While I welcome some disabled people don't think this language is all that important to 'cleanse' as your creepily put it, you will find other disabled people will find this language uncomfortable and it's concerning that you've closed this without inviting anymore perspectives.

I'm not looking to debate you and find your message to be unusually hostile to discussing this. Suggesting it is Orwellian to consider how disabled people feel about language is very strange.

**Reply by Myndex** [reply]
I’m open to discuss or debate this topic, but this is not the venue, as the topic is unrelated to vision. I’m working on an article on the general topic under my AngryAndrew.com pen name, tho it hasn’t published yet, I’d be open to discussion there or in the progressive politics forum when we get that up and running.
